docs/UserGuide.md

```diff
@@ -97,12 +102,13 @@ This can be used after a `find` command to return the list to its default state
 
 ### Finding eateries by name : `find`
 
-_Search for eateries with names that match the keywords._
+_Search for eateries with names that match the keywords.<br>
+You can also choose to randomize a list of eateries to decide on where to eat._
 
 **Format:** `find NAME…​ [-r NUMBER] [-h]`
 
 <div markdown="span" class="alert alert-primary"> :bulb:  **Tip:**
-You can leave NAME empty if you are using the wildcard (-r) feature
+You can leave NAME empty if you are using the randomizer (-r) feature
 </div>
 
 **Arguments:** </br>
```

Comment:
Should be ```<br>``` instead of ```</br>```.
Also need to leave a space between the last asterisk and the ```<br>```, otherwise Jekyll renders it out as text.